R/set_tag.R

```diff
@@ -1,14 +1,16 @@
-#' Set a New Tag and Create a Release
+#' Set a New Tag
 #'
 #' This function only works when run in a GitHub Action on the master branch.
 #' Otherwise it will only return a message.
-#' @export
+#' It sets new tag a the current commit using the related entry from `NEWS.md`
```

Comment:
Just a guess to complete the sentence, please check carefully if this is what you meant to say...
```suggestion
#' It sets a new tag at the current commit using the related entry from `NEWS.md`
```